### PR TITLE
Added python subtypes

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
demisto/etc#23233
## Description
Added `subtype: python2\3` to scripts and integration yml files

Found all yml files with `type: python` and without subtype using
```
for line in $(ack "type: python$" --yaml -l)                                                                                                                                        
do
if grep -q "subtype: python[23]" $line; then continue ;else
echo "$line" >> no_subtype_files ; fi
done
```
Extracted the python subtype by the following scenario:
if the yml has a dockerimage attribute: the subtype will be according to the result of running `python --version` in that image, else the subtype is 2.

### CODE REVIEW

File: Beta_Integrations/Telegram/Telegram.yml [hunk 1]
```diff
@@ -44,4 +44,5 @@ script:
   runonce: false
   script: '-'
   type: python
+  subtype: python2
```
Comment (Itay4) [inline Beta_Integrations/Telegram/Telegram.yml:47]:
```suggestion
  subtype: python3
```

File: Packs/Legacy/Integrations/Confluence/Confluence.yml [hunk 2]
```diff
@@ -29,6 +29,7 @@ configuration:
 script:
   script: ''
   type: python
+  subtype: python3
```
Comment (Itay4) [inline Packs/Legacy/Integrations/Confluence/Confluence.yml:32]:
```suggestion
  subtype: python2
```

File: Packs/Legacy/Integrations/EclecticIQ/EclecticIQ.yml [hunk 3]
```diff
@@ -60,6 +60,7 @@ configuration:
 script:
   script: ''
   type: python
+  subtype: python3
```
Comment (Itay4) [inline Packs/Legacy/Integrations/EclecticIQ/EclecticIQ.yml:63]:
```suggestion
  subtype: python2
```

File: Packs/Legacy/Integrations/GoogleCloudStorage/GoogleCloudStorage.yml [hunk 4]
```diff
@@ -515,5 +515,6 @@ script:
   runonce: false
   script: '-'
   type: python
+  subtype: python2
```
Comment (Itay4) [inline Packs/Legacy/Integrations/GoogleCloudStorage/GoogleCloudStorage.yml:518]:
```suggestion
  subtype: python3
```

File: Packs/Legacy/Integrations/IntezerV2/IntezerV2.yml [hunk 5]
```diff
@@ -125,5 +125,6 @@ script:
   runonce: false
   script: '-'
   type: python
+  subtype: python2
```
Comment (Itay4) [inline Packs/Legacy/Integrations/IntezerV2/IntezerV2.yml:128]:
```suggestion
  subtype: python3
```

**Comment by Itay4** [comment]
lets check by docker image instead

File: Beta_Integrations/Telegram/CHANGELOG.md [hunk 6]
```diff
@@ -1,2 +1,3 @@
 ## [Unreleased]
+-
 Updated the integration's image.
```
Comment (Itay4) [inline Beta_Integrations/Telegram/CHANGELOG.md:3]: something is wrong with the release notes here

File: Packs/Legacy/Integrations/EclecticIQ/EclecticIQ.yml [hunk 7]
```diff
@@ -17,12 +17,12 @@ configuration:
   defaultvalue: ""
   type: 9
   required: true
-- display: Trust any certificate (unsecure)
+- display: Trust any certificate (not secure)
   name: insecure
-  defaultvalue: "true"
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/EclecticIQ/EclecticIQ.yml:22]:
```suggestion
```

File: Packs/Legacy/Integrations/Jask/Jask.yml [hunk 8]
```diff
@@ -27,7 +27,7 @@ configuration:
   name: insecure
   required: false
   type: 8
-- defaultvalue: 'true'
+- defaultvalue: ''
```
Comment (Itay4) [inline Packs/Legacy/Integrations/Jask/Jask.yml:30]:
```suggestion
```

File: Packs/Legacy/Integrations/MaxMind_GeoIP2/MaxMind_GeoIP2.yml [hunk 9]
```diff
@@ -104,6 +105,18 @@ script:
     - contextPath: MaxMind.City
       description: City the IP is located in
       type: string
+    - contextPath: DBotScore.Indicator
```
Comment (Itay4) [inline Packs/Legacy/Integrations/MaxMind_GeoIP2/MaxMind_GeoIP2.yml:108]: if we're not adding it to the code, lets remove it from here

File: Packs/Legacy/Integrations/RTIR/RTIR.yml [hunk 10]
```diff
@@ -11,12 +11,12 @@ configuration:
   name: credentials
   required: true
   type: 9
-- defaultvalue: 'true'
+- defaultvalue: ''
```
Comment (Itay4) [inline Packs/Legacy/Integrations/RTIR/RTIR.yml:14]:
```suggestion
```

File: Packs/Legacy/Integrations/RTIR/RTIR.yml [hunk 11]
```diff
@@ -11,12 +11,12 @@ configuration:
...
   display: Trust any certificate (not secure)
   name: unsecure
   required: false
   type: 8
-- defaultvalue: 'false'
+- defaultvalue: ''
```
Comment (Itay4) [inline Packs/Legacy/Integrations/RTIR/RTIR.yml:19]:
```suggestion
```

File: Packs/Legacy/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.yml [hunk 12]
```diff
@@ -11,12 +11,12 @@ configuration:
   name: credentials
   required: true
   type: 9
-- defaultvalue: 'false'
-  display: Trust any certificate (unsecure)
+- defaultvalue: ''
```
Comment (Itay4) [inline Packs/Legacy/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.yml:14]:
```suggestion
```

File: Packs/Legacy/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.yml [hunk 13]
```diff
@@ -11,12 +11,12 @@ configuration:
...
+  display: Trust any certificate (not secure)
   name: unsecure
   required: false
   type: 8
-- defaultvalue: 'false'
+- defaultvalue: ''
```
Comment (Itay4) [inline Packs/Legacy/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.yml:19]:
```suggestion
```

File: Packs/Legacy/Integrations/SNDBOX/SNDBOX.yml [hunk 14]
```diff
@@ -20,10 +20,10 @@ configuration:
     required: true
   - display: Use system proxy settings
     name: proxy
-    defaultvalue: "False"
+    defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/SNDBOX/SNDBOX.yml:23]:
```suggestion
```

File: Beta_Integrations/Telegram/CHANGELOG.md (hunk 6 above)
Comment (hod-alpert) [reply Beta_Integrations/Telegram/CHANGELOG.md:3]: What is it?

Comment (Itay4) [reply Beta_Integrations/Telegram/CHANGELOG.md:3]: move `Updated the integration's image.` to not be under `Unreleased`

Comment (hod-alpert) [inline Beta_Integrations/Telegram/CHANGELOG.md:3]: How about this? 
```
## [Unreleased]
Updated the integration's image.
```

Comment (Itay4) [reply Beta_Integrations/Telegram/CHANGELOG.md:3]: lets just delete that line, and keep `-`

File: Beta_Integrations/integration-TrendMicroDDA.yml [hunk 15]
```diff
@@ -2,7 +2,7 @@ commonfields:
   id: Trend Micro Deep Discovery Analyzer Beta
   version: -1
 name: Trend Micro Deep Discovery Analyzer Beta
-display: Trend Micro Deep Discovery Analyzer (Beta)
+display: Trend Micro Deep Discovery Analyzer
```
Comment (anara123) [inline Beta_Integrations/integration-TrendMicroDDA.yml:5]: Why Beta removed? because there is `beta: true`

Comment (hod-alpert) [reply Beta_Integrations/integration-TrendMicroDDA.yml:5]: I get \<file-path\> should NOT contain the substring "beta" in a new beta integration.

**Pull request merged**